sim/validator.py

```diff
@@ -6,209 +6,149 @@
 
 import pandas as pd
 
+import sample_sets
+import utils
 import compare_snps
-from sample import *
-from utils import run
+import sequenced
+import processed
 
-DEFAULT_REFERENCE_PATH = './Mycobacterium_bovis_AF212297_LT78304.fa'
-
-
-def btb_seq(btb_seq_directory, reads_directory, results_directory):
-    run(["bash", "./btb-seq", reads_directory,
-          results_directory], cwd=btb_seq_directory)
+import config
 
 def simulate(
-    output_path,
     samples,
-    reference_path=DEFAULT_REFERENCE_PATH,
-    exist_ok=True
+    genomes_path,
+    reads_path, 
 ):
-    """ Runs a performance test against the pipeline
+    """ Simulate a number of samples
 
         Parameters:
-            output_path (str): Path to output
-            samples (list of Sample objects): Samples for building simulated reads
-            reference_path (str): Path to reference genome
-            exist_ok (bool): Whether or not to throw an error if a results directory already exists
+            samples (list of Sample objects): Samples for generating simulated reads
+            genomes_path (string): path to directory containing output genomes
+            reads_path (string): path to directory containing output fastq reads
 
         Returns:
             simulate_reads_path (str): Path to simulated reads 
     """
 
     # Validate Input
```

Comment:
Maybe remove this comment as ehese next lines aren't validating input

Reply:
changed to `Initialise`